src/adapters/ip_camera_adapter/mod.rs

```diff
@@ -98,12 +105,12 @@ struct IpCameraService<T> {
     properties: ServiceProperties,
     ip: String,
     snapshot_dir: String,
-    name : String,
-    model: String,
```

Comment:
I haven't noticed that we use ```model``` field anywhere, so moved it to custom properties, like I did fir hue.